mne/time_frequency/tfr.py

```diff
@@ -1049,10 +1049,9 @@ def to_data_frame(self, picks=None, index=None, long_format=False,
         # check pandas once here, instead of in each private utils function
         pd = _check_pandas_installed()  # noqa
         # arg checking
+        valid_index_args = ['time', 'freq']
         if isinstance(self, EpochsTFR):
-            valid_index_args = ['time', 'freq', 'epoch', 'condition']
-        else:
-            valid_index_args = ['time', 'freq']
+            valid_index_args.append(['epoch', 'condition'])
```

Comment:
I broke your tests; I think this will fix them:

```suggestion
            valid_index_args.extend(['epoch', 'condition'])
```

Reply:
🙏 thx